js/activity.js

```diff
@@ -5931,9 +5931,11 @@ class Activity {
                 console.log("dom is ready");
                 const urlParams = new URLSearchParams(window.location.search);
                 const room_id = urlParams.get('roomID');
+                const collabUrl = window.location.href;
  
                 if (room_id) {
                     this.room_id = room_id;
+                    this.textMsg(`Collaboration link ${collabUrl}`);
```

Comment:
please add _() so that the string is available for translation, e.g.,

this.textMsg(`_("Collaboration link: ") ${collabURL}`)

Reply:
@walterbender I think I have fixed it. Please have a look again.